faasmcli/faasmcli/tasks/codegen.py

```diff
@@ -51,7 +51,3 @@ def local(ctx, wamr=False):
     _do_codegen_user("mpi", wamr=wamr)
     _do_codegen_user("omp", wamr=wamr)
     _do_codegen_user("python", wamr=wamr)
-
```

Comment:
I can't see where these last two lines have moved to. We need to run the _wamr_ codegen for these two functions to run the WAMR tests

Reply:
oh man, thanks for spotting this, I completely overlooked the `wamr` flag :facepalm: